source.lua

```diff
@@ -366,6 +366,15 @@ local function stm_inst_list(S)
 			data.sBx = bit.band(bit.rshift(ins, 14), 0x3FFFF) - 131071
 		end
 
+		if op == 10 then -- decode NEWTABLE array size
```

Comment:
Ideally this check should be moved into the `args == 'ABC'` branch, as to avoid checking it for irrelevant instructions.